install_microshift.py

```diff
@@ -0,0 +1,120 @@
+import subprocess
+import time
+
+def microshift_iso_builder():
+    cmd = f"dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion firewalld" 
+    output = subprocess.run(cmd, shell=True)
+    
+    cmd = f"systemctl enable osbuild-composer.socket cockpit.socket --now" 
+    output = subprocess.run(cmd, shell=True)
+    
+    uname_m= subprocess.check_output(['uname', '-m']).decode('utf-8').strip()
+
+    content = f'''id = "rhocp-4.14"\nname = "Red Hat OpenShift Container Platform 4.14 for RHEL 9"\ntype = "yum-baseurl"\nurl = "https://cdn.redhat.com/content/dist/layered/rhel9/{uname_m}/rhocp/4.14/os"\ncheck_gpg = true\ncheck_ssl = true\nsystem = false\nrhsm = true'''
+
+    with open('rhocp-4.14.toml', 'w') as file:
+        file.write(content)
+
+    content = f'''id = "fast-datapath"\nname = "Fast Datapath for RHEL 9"\ntype = "yum-baseurl"\nurl = "https://cdn.redhat.com/content/dist/layered/rhel9/{uname_m}/fast-datapath/os"\ncheck_gpg = true\ncheck_ssl = true\nsystem = false\nrhsm = true'''
+
+    with open('fast-datapath.toml', 'w') as file:
+        file.write(content)
+
+    cmd = f"sudo composer-cli sources add rhocp-4.14.toml"
```

Comment:
composer-cli has state? shouldn't we start by "purge everything" to make the script idempotent.

Reply:
I agree